samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml

```diff
@@ -38,13 +38,21 @@ spec:
       - name: BUILDKITD_FLAGS
         value: --oci-worker-no-process-sandbox
       command:
-        - buildctl-daemonless.sh
+        - /bin/ash
```

Comment:
any specific reasons on why the usage of alchemist shell?

Reply:
I randomly picked one after I noticed that bash was not available.